trie/zk_trie_test.go

```diff
@@ -25,17 +25,12 @@ import (
 	"sync"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-
 	zkt "github.com/scroll-tech/zktrie/types"
+	"github.com/stretchr/testify/assert"
```

Comment:
It's a nitpick that this can be put above "scroll-tech" headers, but not sure if it's a common practice.